src/pudl/transform/eia.py

```diff
@@ -1340,5 +1341,6 @@ def finished_eia_asset(**kwargs) -> pd.DataFrame:
         "core_eia860__assn_boiler_cooling": "_core_eia860__boiler_cooling",
         "core_eia860__assn_boiler_stack_flue": "_core_eia860__boiler_stack_flue",
         "core_eia860__yearly_generators_wind": "_core_eia860__generators_wind",
+        "core_eia860__yearly_generators_solar": "_core_eia860__generators_solar",
```

Comment:
I think this tale can probably entirely fit into the `core_eia860__scd_generators_solar` camp. In general I would like to avoid if possible breaking up these new little tables on the basis of `yearly` and `scd` if possible.